lang/src/org/partiql/lang/eval/visitors/GroupByPathExpressionVisitorTransform.kt

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package org.partiql.lang.eval.visitors
+
+import org.partiql.lang.ast.IsSyntheticNameMeta
+import org.partiql.lang.ast.UniqueNameMeta
+import org.partiql.lang.ast.metaContainerOf
+import org.partiql.lang.ast.toIonElementMetaContainer
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.domains.PartiqlAst.Builder.caseInsensitive
+import org.partiql.lang.eval.errNoContext
+
+/**
+ * This rewrite must execute after [GroupByItemAliasVisitorTransform] and [FromSourceAliasVisitorTransform].
```

Comment:
**Reminder** for when merging `visitor-transform` into master: add these visitor transforms as imports so the kdoc works correctly.